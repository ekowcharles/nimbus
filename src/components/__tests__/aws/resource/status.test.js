import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ResourceStatus from '../../../aws/resource/status'

describe('ResourceStatus', () => {
  it('renders status or resource', () => {
    const wrapper = shallow(
      <ResourceStatus
        hide={false}
        state="available"
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})