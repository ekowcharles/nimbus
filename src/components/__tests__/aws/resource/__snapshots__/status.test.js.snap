// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResourceStatus renders status or resource 1`] = `
<Fragment>
  <div
    className="res-status-tooltip"
  >
    <div
      className="res-status available"
    />
    <span
      className="text"
    >
      available
    </span>
  </div>
</Fragment>
`;
